project/plugins.sbt

```diff
@@ -17,6 +17,7 @@ addSbtPlugin("com.lightbend.paradox" % "sbt-paradox" % "0.6.8")
 addSbtPlugin("io.github.jonas" % "sbt-paradox-material-theme" % "0.6.0")
 addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.3.5")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.1.1")
+addSbtPlugin("com.github.cb372" % "sbt-explicit-dependencies" % "0.2.12")
```

Comment:
This change just slipped through by mistake 😢

Reply:
Reverted it here https://github.com/spotify/scio/commit/4d731cab98ee13887c407e7a915ac54e2e8aac5a